spec/paperclip/io_adapters/registry_spec.rb

```diff
@@ -15,6 +15,19 @@ def initialize(target); end
     end
   end
 
+  context "register" do
+    it "allows overriding" do
+      class OriginalAdapter; def initialize(target);end; end
+      class NewAdapter; def initialize(target);end; end
```

Comment:
Unused method argument - `target`. If it's necessary, use `_` or `_target` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.<br>Use empty lines between defs.<br>Space missing after semicolon.